pkg/gateway/handler.go

```diff
@@ -181,9 +182,11 @@ func RepoOperationHandler(sc *ServerContext, handler operations.RepoOperationHan
 		repoOperation := &operations.RepoOperation{
 			AuthorizedOperation: authOp,
 			Repository:          repo,
+			MatchedHost:         matchedHost,
 		}
 		req = req.WithContext(logging.AddFields(ctx, logging.Fields{
-			"repository": repo.Name,
+			"repository":   repo.Name,
+			"matched_host": matchedHost,
```

Comment:
(Not in this PR, but... we really need to document the added context fields somehow and somewhere.Maybe we should make the key a named string type,and use constants.

Reply:
true, adding I'll describe it in an issue - at least the one we pass through the context, the keys can be part of our logging package is it is a wrapper around logrus